plugins/crashes/frontend/public/javascripts/countly.views.js

```diff
@@ -369,12 +526,41 @@
         },
         methods: {
             refresh: function() {
-                return this.$store.dispatch("countlyCrashes/crash/refresh");
+                var promises = [];
+                var self = this;
+
+                promises.push(this.$store.dispatch("countlyCrashes/crash/refresh"));
+
+                if (this.symbolicationEnabled) {
+                    promises.push(new Promise(function(resolve, reject) {
+                        countlyCrashSymbols.fetchSymbols(true)
+                            .then(function(symbolIndexing) {
+                                self.symbols = {};
+
+                                var buildIdMaps = Object.values(symbolIndexing);
+                                buildIdMaps.forEach(function(buildIdMap) {
+                                    Object.keys(buildIdMap).forEach(function(buildId) {
+                                        self.symbols[buildId] = buildIdMap[buildId];
+                                    });
+                                });
+
+                                resolve(this.symbols);
+                            }).catch(function(err) {
+                                reject(err);
```

Comment:
If this catch is for a request which was initiated by CV.$.ajax, then you should make sure that you pass `disableAutoCatch: true` as option. Otherwise it wouldn't be caught by this one. You just reject it, though. Just a note.